SFR-1410_ManifestScript.py

```diff
@@ -0,0 +1,68 @@
+import boto3
+import json
+import logging
+
+from urllib.parse import urlparse, parse_qsl, urlencode, urlunparse
+
+logging.basicConfig(filename='manifest.log', encoding='utf-8', level=logging.INFO)
```

Comment:
I didn't focus on this before but this is a good logging implementation. There is a shared DRB logging implementation, but we might want to update this to output to a (rotated) log file as you do here. A potential improvement for the future we should remember.